src/raid.cpp

```diff
@@ -767,6 +771,27 @@ bool RaidBufferManager::detectSlowestRaidConnection(unsigned thisConnection, uns
     return false;
 }
 
+
+m_off_t RaidBufferManager::progress()
+{
+    assert(isRaid());
+    m_off_t reportPos = 0;
+
+    for (unsigned j = RAIDPARTS; j--; )
```

Comment:
Still think a reverse loop should only be used if there's a reason for it. Otherwise might confuse the reader...

Reply:
I think anyone familiar with our code base knows about this one :-)  It is a very nice, very concise method once you're used to it

Reply:
I tend to agree with Christian here, but let's leave it as it is :)